src/lemonade_server/server_models.json

```
@@ -318,5 +318,11 @@
         "recipe": "llamacpp",
         "suggested": true,
         "labels": ["hot", "reasoning"]
+    },
+    "GLM-4.5-Air-UD-Q4K-XL-GGUF": {
+        "checkpoint": "unsloth/GLM-4.5-Air-GGUF:UD-Q4_K_XL",
+        "recipe": "llamacpp",
+        "suggested": false,
+        "labels": ["reasoning"]
```

I think Jeremy wanted you to use `"labels": ["reasoning","hot"]` not `"labels": ["reasoning", ”hot”]`

```suggestion
        "labels": ["reasoning","hot"]
```

---

The space? the model directly above has a space... `"labels": ["hot", "reasoning"]`